subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r36/BuildProgressCrossVersionSpec.groovy

```diff
@@ -73,4 +76,37 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
         events.operation('Task :runInProcess').descendant('My in-process worker action')
         events.operation('Task :runForked').descendant('My forked worker action')
     }
+
+    def "when running a build then root build progress event is 'Run build'"() {
```

Comment:
I think we already have these test cases. What we _are_ missing is some test coverage that ensures there are similar events when running Gradle from the command-line